src/components/services/WalletLifecycle.js

```diff
@@ -103,13 +107,12 @@ export class WalletLifecycleComponent extends React.Component<Props> {
       return true
     })
 
-    // If the booting list has < 3 items, boot some wallets:
-    for (const walletId of activeWalletIds) {
-      if (this.booting.length >= 3) break
-
-      // Ignore missing wallets, started wallets, and already-booting wallets:
-      const wallet = currencyWallets[walletId]
-      if (wallet == null || guiWallets[walletId] == null) continue
+    // Use the sortedWalletList to boot the wallets in the same order they appear in the list
+    for (const walletItem of sortedWalletList) {
+      if (this.booting.length >= BOOT_LIMIT) break
+      const { token, tokenId, wallet, walletId } = walletItem
+      // Ignore missing wallets, token rows, started wallets, and already-booting wallets:
```

Comment:
I like a blank line before each commented "paragraph" of code.

Reply:
done